src/app/common/components/KeyDisplayIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { Card } from '@rebass/emotion';
+import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import theme from '../../../theme';
+import * as React from 'react';
+
+interface IProps {
+  isHidden: boolean;
+  id: string;
+}
+
+const KeyDisplayIcon: React.FunctionComponent<IProps> = ({
+  isHidden,
+  id,
+  ...rest
+}) => {
+  const HiddenIcon = styled(EyeSlashIcon)`
```

Comment:
look into fixing positioning of this icon. Is there a way to add it to be sitting inside of the field?